client/src/styles/nvidia-theme.scss

```
@@ -1581,6 +1581,13 @@ body {
   font-size: var(--text-sm);
 }
 
+.status-value--wrap {
+  min-width: 0;
+  text-align: right;
+  white-space: normal;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)